pkg/collector/corechecks/system/iostats_pdh_windows.go

```diff
@@ -63,7 +65,7 @@ func init() {
 func isDrive(instance string) bool {
 	found := false
 	instance += "\\"
-	if instance != "C:\\" {
+	if !drivePattern.MatchString(instance) {
```

Comment:
I have no idea what I was thinking there... 

I think the regex needs to be a bit more liberal... I don't know in what instances it's capital,and what instances is lowercase, but drives `c:` == `C:`.

When I look at the available drives on my system,  I have
\LogicalDisk(HarddiskVolume1)
\LogicalDisk(C:)
\LogicalDisk(HarddiskVolume4)
\LogicalDisk(HarddiskVolume5)
\LogicalDisk(D:)
\LogicalDisk(J:)
\LogicalDisk(HarddiskVolume101)
\LogicalDisk(_Total)

The "HarddiskVolumeX" are duplicates of the mounted drives (in this case, c,d & j) however, I believe that they don't have to be (for instance, in cases when drives are mounted on different mount points or not at all).  May want to look into this a bit deeper.

Reply:
In A5 it seems that we just sent everything except `_Total` : 
https://github.com/DataDog/dd-agent/blob/bfc3057fa68b2d6bbc223780c3933e893a6ba38f/checks/system/win32.py#L29-L31

We should probably keep A5 behavior. That said data tagged with `HarddiskVolumeX` would probably be hard to use / duplicate in most cases.

WDYT @derekwbrown  @olivielpeau ?

Reply:
I'm fine with reporting only mounted "lettered" drives, at least for 6.3.

We can clarify later if these `HarddiskVolume` drives make sense to report. The Agent 5 logic seems to report them indeed, but I don't think we ever really thought about them so it's OK to revisit now.

How does that sound @derekwbrown?